apps/wakunode2/app.nim

```diff
@@ -136,44 +130,14 @@ proc init*(T: type App, rng: ref HmacDrbgContext, conf: WakuNodeConf): T =
   )
 
 
-## SQLite database
-
-proc setupDatabaseConnection(dbUrl: string): AppResult[Option[SqliteDatabase]] =
-  ## dbUrl mimics SQLAlchemy Database URL schema
-  ## See: https://docs.sqlalchemy.org/en/14/core/engines.html#database-urls
-  if dbUrl == "" or dbUrl == "none":
-    return ok(none(SqliteDatabase))
-
-  let dbUrlParts = dbUrl.split("://", 1)
-  let
-    engine = dbUrlParts[0]
-    path = dbUrlParts[1]
-
-  let connRes = case engine
-    of "sqlite":
-      # SQLite engine
-      # See: https://docs.sqlalchemy.org/en/14/core/engines.html#sqlite
-      SqliteDatabase.new(path)
-
-    else:
-      return err("unknown database engine")
-
-  if connRes.isErr():
-    return err("failed to init database connection: " & connRes.error)
-
-  ok(some(connRes.value))
-
-
 ## Peer persistence
 
-const PeerPersistenceDbUrl = "sqlite://peers.db"
-
 proc setupPeerStorage(): AppResult[Option[WakuPeerStorage]] =
-  let db = ?setupDatabaseConnection(PeerPersistenceDbUrl)
+  let db = ? SqliteDatabase.new("peers.db")
```

Comment:
Nit: Better to extract this to a `const` just before the function than to make this a "magic value" inline with the code: easier to debug; clear where to add config if needed.